doc/plugins/eu.esdihumboldt.hale.doc.user/html/getting_started/install_plugins.html

```diff
@@ -7,16 +7,23 @@
 <body>
 	<h1>Installing plugins</h1>
 	<p>
-		Since version 3.5, hale studio supports the installation of third-party plugins
-		via so-called update sites. These update sites can be hosted on the internet
+		Since version 3.5, hale studio supports the installation of plugins
+		via update sites. Update sites can be hosted on the internet
 		or reside on your local disk.
-		
+	</p>
+  <p>
+		hale studio connectivity plugins enable users to directly access data in Oracle databases and ESRI File Geodatabases
+		using the Oracle Database Reader/Writer and the FileGeodatabase Reader. The connectivity plugins are available to users with a
+		<a href="https://www.wetransform.to/products/halestudiopro/">hale studio pro package</a>. For more information, please contact
+		wetransform at <b>info@wetransform.to</b>.
+  </p>
+	<p>
 		To install a new hale studio plugin, open the software installation dialog
 		via the <code>Help|Install new software...</code> menu item and select the
 		appropriate update site from the list.
 	</p>
 	<div>
-		<img src="software_installation_dialog.png" title="Software installation dialog"
+		<img src="software_installation_dialog_updated.png" title="Software installation dialog"
```

Comment:
I'd suggest to just replace the old image files instead of adding new ones with an `_updated` suffix. I see no reason to keep the old ones around if they're not used anymore.

Reply:
OK, thanks for the tip. I wasn't sure if we wanted to archive them, or something. I was hesitant to just delete them. Should I go back and delete all the screenshots I did this for, or take this approach going forward?